src/components/greetings.jsx

```diff
@@ -0,0 +1,20 @@
+function Greetings({ lang, children }) {
+    let message;
+    if (lang === 'en') {
+        message = "Hellow";
+    }
+    if (lang === 'de') {
+        message = "Hallo";
+    }
+    if (lang === 'es') {
+        message = "Hola";
+    }
+    if (lang === 'fr') {
+        message = "Bonjour";
+    }
```

Comment:
Mira otra forma de hacerlo, en vez de generar la estructura de IFs (que esta de perfect!) vamos a hacerlo de otra forma, tiene sentido que tengas un arbol para especificar esto no? por ejemplo en un diccionario vas a buscas, pues vamos a resolverlo de esa forma, te generas un objeto y que las claves sean lo que te pasan en la prop
```suggestion
   const message = { es: "Hola", de: "Hallo", en: "Hello", fr: "Bonjout" };
```

Entonces despues en el message lo unico que tendrias que poner es {message[lang]} y lo tienes!!

Piensa que esto seria mas escalable, ya que si quieres añadir un idioma nuevo, lo unico que tendrias que hacer es añadir una clave con su valor, y no toda la estructura de IF (que es un toston)